src/components/light.js

```diff
@@ -46,10 +46,37 @@ module.exports.Component = registerComponent('light', {
       // Light type has not changed. Update light.
       Object.keys(diffData).forEach(function (key) {
         var value = data[key];
-        if (['color', 'groundColor'].indexOf(key) !== -1) {
-          value = new THREE.Color(value);
+
+        switch (key) {
+          case 'color':
```

Comment:
they were being set to a new THREE.Color, since i changed the key checks to a switch, i set it there. I guess that is to convert color values?

I'll add the curly braces.